nixos/modules/virtualisation/amazon-config.nix

```diff
@@ -1,5 +1,3 @@
-{ config, pkgs, modulesPath, ... }:
-
 {
-  imports = [ "${modulesPath}/virtualisation/amazon-image.nix" ];
+  imports = [ ./amazon-image.nix ];
```

Comment:
I fixed this as you suggested in commit 9647a72835f1fc0ef62fb25f249b5631c5869431